testData/solutions/win32.json

```diff
@@ -12,13 +12,49 @@
             {
                 "type": "gpii.settingsHandlers.INISettingsHandler",
                 "options": {
-                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\16.0\\Settings\\enu\\DEFAULT.JCF",
```

Comment:
Here are the exact locations in JAWS 16
"${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\16.0\\Settings\\enu\\DEFAULT.JCF" (Main settings, you can select the default voice profile here. I always set the voice profile to GPII.VPF and create a VPF file)
"${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\16.0\\Settings\\VoiceProfiles\\GPII.VPF"
(This is the voice profile, GPII.VPF, you can create your own voice profile, *name*.VPF)